src/cascadia/TerminalSettingsEditor/Appearances.cpp

```diff
@@ -42,6 +43,143 @@ namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
         return _hasPowerlineCharacters.value_or(false);
     }
 
+    Windows::Foundation::Collections::IMap<winrt::hstring, winrt::hstring> Font::FontAxesTagsAndNames()
+    {
+        auto fontAxesTagsAndNames = winrt::single_threaded_map<winrt::hstring, winrt::hstring>();
+
+        winrt::com_ptr<IDWriteFont> font;
+        THROW_IF_FAILED(_family->GetFont(0, font.put()));
+        winrt::com_ptr<IDWriteFontFace> fontFace;
+        THROW_IF_FAILED(font->CreateFontFace(fontFace.put()));
+        winrt::com_ptr<IDWriteFontFace5> fontFace5{ fontFace.as<IDWriteFontFace5>() };
```

Comment:
nit: I'd suggest replacing all these `winrt::com_ptr` with `wil::com_ptr` as the WIL helpers are more powerful and easier (and safer) to use. If you do choose to replace these, you'd also have to do it in other parts of this PR.

In either case though, the `.as` must be replaced with a `.try_as` (wil: `.try_query`), because `IDWriteFontFace5` is a very recent interface (build 20348).